samples/firefox-accounts/src/main/java/org/mozilla/samples/fxa/LoginFragment.kt

```diff
@@ -0,0 +1,87 @@
+package org.mozilla.samples.fxa
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Bitmap
+import android.net.Uri
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.webkit.CookieManager
+import android.webkit.WebView
+import android.webkit.WebViewClient
+
+private const val AUTH_URL = "authUrl"
+private const val REDIRECT_URL = "redirectUrl"
+
+class LoginFragment : Fragment() {
```

Comment:
Have a look at `WebViewFragment`: https://android.googlesource.com/platform/frameworks/base/+/refs/heads/master/core/java/android/webkit/WebViewFragment.java

You should forward the lifecycle events from the fragment to the WebView. Unfortunately WebViewFragment is deprecated so we can't just extend it.

Reply:
The android docs suggest that only onResume() and onPause() need to be manually called, but the original WebViewFragment also overrides some other methods (ie. onDestroy(), onDestroyView()) in addition to those two. Is that necessary, given that OnCreateView also preemptively destroys the view?

Reply:
[destroying webview on create](https://github.com/mozilla-mobile/android-components/pull/438/commits/776c5e9ff7a40cccd5dc29e68d5cbc2fee9508fb#diff-4c80ae4600b3139465e587ba9871855eR61)
[lifecycle methods](https://github.com/mozilla-mobile/android-components/pull/438/commits/776c5e9ff7a40cccd5dc29e68d5cbc2fee9508fb#diff-4c80ae4600b3139465e587ba9871855eR82)